com.unity.render-pipelines.high-definition/Editor/Material/MaterialHeaderScope.cs

```diff
@@ -18,8 +18,9 @@ namespace UnityEditor.Rendering.HighDefinition
     /// </code>
     /// </example>
     /// </summary>
-    internal struct MaterialHeaderScope : IDisposable
+    public struct MaterialHeaderScope : IDisposable
     {
+        /// <summary>True if the header expanded. False otherwise.</summary>
```

Comment:
Indicates whether the header is expanded or not. Is `true` if the header is expanded, `false` otherwise.